server/src/pages/homepage/model.js

```diff
@@ -1,7 +1,8 @@
 exports.createModel = function(req) {
   let model = {
     title: "Firefox Screenshots",
-    showMyShots: !!req.deviceId
+    showMyShots: !!req.deviceId,
+    userAgent: req.headers['user-agent']
```

Comment:
I think it would be better here to do:

```js
let isFirefox = (/firefox\/\d{1,255}/i).test(req.headers['user-agent'])
```

and then put that in as a property of the model. Then the test is simpler in the view.